package.json

```diff
@@ -23,7 +23,7 @@
     "@fortawesome/vue-fontawesome": "^0.1.6",
     "@molgenis/molgenis-api-client": "^3.1.0",
     "@molgenis/molgenis-i18n-js": "^0.1.1",
-    "@molgenis/molgenis-ui-context": "^1.4.0",
+    "@molgenis/molgenis-ui-context": "^1.4.1",
```

Comment:
De version pinning staat minor updates toe. Zo te zien heeft die 1.4.2 geinstalleerd. In plaats van caret/tilde ranges kun je ook x-ranges gebruiken(https://docs.npmjs.com/misc/semver). Bij het gebruik van de caret/tilde ranges moet ik zelf altijd weer even opzoeken hoe het zat. Met x-ranges vind ik dat zelf wat intuitiever. Dat leest dan als:
`"@molgenis/molgenis-ui-context": "1.4.x",`